garage/tf/samplers/off_policy_vectorized_sampler.py

```diff
@@ -0,0 +1,155 @@
+import itertools
+import pickle
+
+import numpy as np
+
+from garage.misc import tensor_utils
+from garage.misc.overrides import overrides
+from garage.replay_buffer.base import Buffer
+from garage.tf.envs import VecEnvExecutor
+from garage.tf.samplers import BatchSampler
+
+
+class OffPolicyVectorizedSampler(BatchSampler):
```

Comment:
Why do we need two samplers for on policy and off policy? I believe that these two could be easily unified.

Reply:
A better question is "which parts are the same and which parts are different?"